CodeGen/Generators/UnitsNetWrcGenerator.cs

```diff
@@ -52,6 +52,7 @@ public static void Generate(string rootDir, Quantity[] quantities)
 
             Log.Information("");
             GenerateUnitAbbreviationsCache(quantities, $"{outputDir}/UnitAbbreviationsCache.g.cs");
+            GenerateUnitSingularNamesCache(quantities, $"{outputDir}/UnitSingularNamesCache.g.cs");
```

Comment:
As mentioned, I believe you can revert this entire file. Windows Runtime Component build target is no longer receiving new features.